k-distribution/tests/pyk/Makefile

```diff
@@ -33,11 +33,17 @@ kompile: $(llvm_imp_kompiled) $(haskell_imp_kompiled)
 
 # Virtualenv
 
-VENV     := venv/pyvenv.cfg
-ACTIVATE := . venv/bin/activate
+VENV        := venv/pyvenv.cfg
+ACTIVATE    := . venv/bin/activate
+PYTHON      := python3
+PIP         := pip
+PIP_VERSION := 22.0.4
 
 $(VENV):
-	virtualenv -p python3.8 venv && $(ACTIVATE) && pip install -e $(K_LIB)/pyk
+	   $(PYTHON) -m venv venv                 \
+	&& $(ACTIVATE)                            \
+	&& $(PIP) install --upgrade $(PIP)        \
+	&& $(PIP) install --editable $(K_LIB)/pyk
```

Comment:
To get this working, I had to use my global pip to do `pip install --upgrade pip`, which made a user-local install of pip I think.

Then I had to remove my global install of pip, and remove my global install of virtualenv. Then do `pip install virtualenv` to make a user install. Then this change make it work on my system (because now I have pip22.0.4).

Reply:
So this fix didn't work: https://github.com/runtimeverification/k/pull/2467/commits/1fa69386abdf734414437c011817ff569110750d?

Reply:
I've backed out these changes.